flake8_isort.py

```diff
@@ -1,4 +1,7 @@
 # -*- coding: utf-8 -*-
+from flake8_polyfill import stdin
+stdin.monkey_patch('pycodestyle')
```

Comment:
Unfortunately the import order is important, if we keep the other imports as they are; the monkeypatch must happen before we `from pycodestyle import stdin_get_value` or else the monkeypatch does not take effect.

I just realized, however, that if we instead just `import pycodestyle` and then later in the code use `pycodestyle.stdin_get_value()`, we can do the monkeypatch after the import and it works. I can update the PR to use that approach if you prefer.